tests/unit/test_interpolation_tables.py

```diff
@@ -58,47 +59,155 @@ def diff(self, x, axis):
 
 @pytest.mark.parametrize("dim", [2, 3, 4])
 @pytest.mark.parametrize("factory", [_LinearFunc, _QuadraticFunc])
-def test_interpolation(dim, factory):
+def test_value_assigment_and_interpolation(dim, factory):
```

Comment:
Perhaps for the sake of clarity, the test `test_value_assigment_and_interpolation` can be split into 
- test_value_assigment
- test_interpolation
- test_adaptive_interpolation

Reply:
Fair point, although I'll do a slightly different splitting.